client/src/js/subtraction/components/Detail/__tests__/File.test.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { SubtractionFile } from "../File";
+import { byteSize } from "../../../../utils/utils";
+
+describe("<SubtractionFile />", () => {
+    let props = {};
+
+    beforeEach(() => {
+        props.file = {
+            download_url: "/api/subtractions/xl8faqqz/files/subtraction.fa.gz",
+            id: 1,
+            name: "foo",
+            size: 36461731,
+            subtraction: "xl8faqqz",
+            type: "fasta"
+        };
```

Comment:
Please follow pattern used in rest of Virtool tests.
```js
// Declare variable without value.
let props;
```

In `beforeEach()`, declare a value for `prop`:
```js
beforeEach(() => {
    props = {
        file: {}
    }
});
```

Of course, this has to change because you are going to use a more parametrized approach for `SubtractionFile`.